x-pack/plugin/sql/src/test/java/org/elasticsearch/xpack/sql/expression/function/scalar/datetime/DateTimeParseProcessorTests.java

```diff
@@ -130,5 +185,18 @@ public void testParsing() {
                 .asProcessor()
                 .process(null)
         );
+        // TimeParse
+        assertEquals(
+                time(10, 20, 30, 123000000, zoneId),
+                new TimeParse(Source.EMPTY, l("10:20:30.123"), l("HH:mm:ss.SSS"), zoneId).makePipe()
+                    .asProcessor()
+                    .process(null)
+        );
+        assertEquals(
+                time(10, 20, 30, 123456789, ZoneOffset.of("+05:30"), zoneId),
+                new TimeParse(Source.EMPTY, l("10:20:30.123456789 +05:30"), l("HH:mm:ss.SSSSSSSSS zz"), zoneId).makePipe()
+                    .asProcessor()
+                    .process(null)
+        );
```

Comment:
Again, indentation should be consistent. This method should look like below. As it is now, it is not like I commented in my review.
```
    public void testParsing() {
        // DateTimeParse
        ZoneId zoneId = ZoneId.of("America/Sao_Paulo");
        assertEquals(
            dateTime(2020, 4, 7, 10, 20, 30, 123000000, zoneId),
            new DateTimeParse(Source.EMPTY, l("07/04/2020 10:20:30.123"), l("dd/MM/uuuu HH:mm:ss.SSS"), zoneId)
                .makePipe()
                .asProcessor()
                .process(null)
        );
        assertEquals(
            dateTime(2020, 4, 7, 5, 20, 30, 123456789, zoneId),
            new DateTimeParse(Source.EMPTY, l("07/04/2020 10:20:30.123456789 Europe/Berlin"), l("dd/MM/uuuu HH:mm:ss.SSSSSSSSS VV"), zoneId)
                .makePipe()
                .asProcessor()
                .process(null)
        );
        assertEquals(
            dateTime(2020, 4, 7, 1, 50, 30, 123456789, zoneId),
            new DateTimeParse(Source.EMPTY, l("07/04/2020 10:20:30.123456789 +05:30"), l("dd/MM/uuuu HH:mm:ss.SSSSSSSSS zz"), zoneId)
                .makePipe()
                .asProcessor()
                .process(null)
        );
        // TimeParse
        assertEquals(
            time(10, 20, 30, 123000000, zoneId),
            new TimeParse(Source.EMPTY, l("10:20:30.123"), l("HH:mm:ss.SSS"), zoneId).makePipe()
                .asProcessor()
                .process(null)
        );
        assertEquals(
            time(10, 20, 30, 123456789, ZoneOffset.of("+05:30"), zoneId),
            new TimeParse(Source.EMPTY, l("10:20:30.123456789 +05:30"), l("HH:mm:ss.SSSSSSSSS zz"), zoneId).makePipe()
                .asProcessor()
                .process(null)
        );
    }
```